docs/source/tv_tensors.rst

```diff
@@ -1,13 +1,13 @@
-.. _datapoints:
+.. _tv_tensors:
 
-Datapoints
+TVTensors
 ==========
 
-.. currentmodule:: torchvision.datapoints
+.. currentmodule:: torchvision.tv_tensors
 
-Datapoints are tensor subclasses which the :mod:`~torchvision.transforms.v2` v2 transforms use under the hood to
+TVTensors are tensor subclasses which the :mod:`~torchvision.transforms.v2` v2 transforms use under the hood to
 dispatch their inputs to the appropriate lower-level kernels. Most users do not
-need to manipulate datapoints directly and can simply rely on dataset wrapping -
+need to manipulate tv_tensors directly and can simply rely on dataset wrapping -
```

Comment:
Occurrences like this "tv_tensor" one should probably become "TVTensor". But this can be done in a follow-up PR that cleans the docs